src/app/collections/PresentationNode.tsx

```diff
@@ -174,6 +201,22 @@ export default class PresentationNode extends React.Component<Props> {
             </div>
           </div>
         )}
+        {childrenExpanded && presentationNodeHash === 1024788583 && (
+          <div className="presentationNodeOptions">
+            <Checkbox
+              label="Hide Completed"
```

Comment:
Will need to add `i18n`
```suggestion
              label={t('Triumphs.HideCompleted')}
```